src/components/Cluster/ClusterDetail/AddNodePool.js

```diff
@@ -352,8 +446,48 @@ class AddNodePool extends Component {
               value={this.state.aws.instanceType.value}
             />
             <p>{`${CPUCores} CPU cores, ${RAM} GB RAM each`}</p>
+            {this.state.allowAlikeInstances && (
+              <FlexWrapperDiv className='alike-wrapper'>
+                <CheckboxWrapper>
+                  <input
+                    className='original-checkbox'
+                    type='checkbox'
+                    checked={this.state.aws.useAlike}
+                    onChange={this.setUseAlikeInstancesEnabled}
+                    id='aws-alike-instances-enabled'
+                  />
+                </CheckboxWrapper>
```

Comment:
Can this be converted to a generic UI component? We could use a styled checkbox 😬

Reply:
Sure! I can do it in another PR if you don't mind?